detox/test/e2e/03.actions.test.js

```diff
@@ -26,6 +26,41 @@ describe('Actions', () => {
     await driver.tapsElement.assertTappedOnce();
   });
 
+  it('should perform activate accessibility action', async () => {
+    await element(by.id("View7991")).performAccessibilityAction('activate');
+    await expect(element(by.text('Accessibility Action activate Working!!!'))).toBeVisible();
+  });
+
+  it('should perform magicTap accessibility action', async () => {
+    await element(by.id("View7991")).performAccessibilityAction('magicTap');
```

Comment:
Pardon my ignorance, but are those names custom or built-in? I'm not very familiar with this domain in mobile.

I'm just wondering whether we need such extensive coverage or 1-2 tests should be enough?

Reply:
Also you could use `it.each([ ...testcases ...])` construct in Jest API since those tests look too monotonous to my taste.

Reply:
These are the standard actions within React Native https://reactnative.dev/docs/accessibility#accessibility-actions.

Reply:
At a minimum I think we should include one standard action and one custom action, but I think it would be best to include a test for each of the standard actions if possible, so I've kept them all and just changed them to use `it.each` as you suggested.
Please let me know if you disagree and would prefer for me to remove some of the actions.